client/src/components/index.jsx

```diff
@@ -33,51 +33,42 @@ class App extends React.Component {
 
   componentDidMount() {
     var path = window.location.pathname.split('/');
-    
+
     let id = path[1];
     axios(`http://localhost:3003/house/${id}`)
     .then(response => {
       let house = response.data[0];
+      console.log(house);
         this.setState({
           type: house.type,
           yearBuilt: house.year_built,
           factsHeating: house.heating,
           factsCooling: house.cooling,
           parking: house.parking,
-          lot: house.lot
+          lot: house.lot,
+          beds: house.bedrooms,
+          baths: house.bathrooms,
+          heating: house.interiorheating,
+          cooling: house.interiorcooling,
+          kitchen: house.kitchen,
+          appliances: house.appliances,
+          flooring: house.flooring +' sqft'
         });
       })
       .catch( error => {
         console.log(error);
       });
 
-    axios(`http://localhost:3003/house/interior/${id}`)
-      .then(response => {
-        let house = response.data[0];
-          this.setState({
-            beds: house.bedrooms,
-            baths: house.bathrooms,
-            heating: house.interiorheating,
-            cooling: house.interiorcooling,
-            kitchen: house.kitchen,
-            appliances: house.appliances,
-            flooring: house.flooring +' sqft'
-          });
-        })
-        .catch( error => {
-          console.log(error);
-        });
-
   }
   render() {
```

Comment:
Great job with the refactor. This is a suggestion that might help with decreasing character count. Inside the render function you can deconstruct the "this.state" object properties and then reference them inside the "Facts_Features" and "InteriorFeatures" components. Here's an example: 

line 63 1/2 `const { type, yearBuilt, factsCooling, factsHeating, parking, lot} = this.state;`

line 67 ```
<Facts_Features type={type} yearBuilt={yearBuilt} factsCooling={factsCooling}
            factsHeating={factsHeating} parking={parking} lot={lot}/>
```

Again, this is just a suggestion it does not improve performance, but it does "DRY" up the code.

Reply:
Great idea to look into, still not used to doing destructing normally, but should get used to it.